### PR TITLE
Tests fixes and small adjustments

### PR DESCRIPTION
**Pull request by vicenterusso**

### CODE REVIEW

File: packages/isar/pubspec.yaml [hunk 1]
```diff
@@ -14,10 +14,12 @@ environment:
 
 dependencies:
   ffi: ">=2.0.0 <3.0.0"
+  isar_test:
```
Comment (mrclauss) [inline packages/isar/pubspec.yaml:17]: Do you need this extra dependency (for everybody using Isar)?

File: packages/isar_flutter_libs/pubspec.yaml [hunk 2]
```diff
@@ -12,9 +12,8 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
-  isar: 
-    version: 3.1.4
-    hosted: https://isar-community.dev/
+  isar:
```
Comment (mrclauss) [inline packages/isar_flutter_libs/pubspec.yaml:15]: Referring local paths won't work for everybody else

**Review by mrclauss — CHANGES_REQUESTED** [review]
Referring to local paths in pubspec works for your (or the github action) setup, but not for the end user...

**Comment by vicenterusso** [comment]
Do you have a use case where this will not work for end users?

**Comment by mrclauss** [comment]
> Do you have a use case where this will not work for end users?

Uhm for everyone :)
I'm pretty sure you cannot publish a package to pub.dev or our package repo that refers to local paths in its dependencies...

**Comment by vicenterusso** [comment]
> > Do you have a use case where this will not work for end users?
> 
> Uhm for everyone :) I'm pretty sure you cannot publish a package to pub.dev or our package repo that refers to local paths in its dependencies... This will only work if you have the isar sources checked out locally (which the regular user won't want)

Should be an easy change after publishing then

**Comment by richard457** [comment]
we might need to consider https://melos.invertase.dev/ for local packages this can help when it is time to publish as well to pub.dev

**Review by mrclauss — APPROVED** [review]

**Comment by mrclauss** [comment]
Looks good to me

**Pull request merged**